test/e2e/main.go

```diff
@@ -40,7 +40,7 @@ func main() {
 
 	if !specificTestFound {
 		logger.Println("No particular test specified. Running all tests.")
-		logger.Println("go run ./test/e2e <test_name> to run a specific test")
+		logger.Println("make test-e2e <test_name> to run a specific test")
 		logger.Printf("Valid tests are: %s\n\n", getTestNames(tests))
```

Comment:
nit: Does this need to be printed? Doesn't each test usually print their names upon start?